src/modules/verifiable-credentials/verifiable-credentials-base.service.ts

```diff
@@ -169,8 +169,16 @@ export abstract class VerifiableCredentialsServiceBase {
 
     const selections = await Promise.all(
       credentialQuery.map(async ({ presentationDefinition }) => {
+        const presentationDefFiltered = {
+          ...presentationDefinition,
+          ...{
+            input_descriptors: this.filterSelfSignDescriptors(
+              presentationDefinition?.input_descriptors
+            ),
+          },
```

Comment:
Can this be simplified?
```suggestion
            input_descriptors: this.filterSelfSignDescriptors(
              presentationDefinition?.input_descriptors
            ),
```